inductiva/tasks/task.py

```diff
@@ -760,6 +763,48 @@ def _contains_only_std_files(self, output_dir: pathlib.Path) -> bool:
         return all(
             file.name in self.STANDARD_OUTPUT_FILES for file in output_files)
 
+    def _get_download_output_url(
```

Comment:
this is actually not returning an URL but rather a request body. Maybe the name can be made clearer?